tests/cluster/delta/deltaspec/tables.py

```diff
@@ -0,0 +1,41 @@
+from spetlr.deltaspec.DeltaTableSpec import DeltaTableSpec
+
+# Note that the table propery delta.columnMapping.maxColumnId is completely ignored
+
+base = DeltaTableSpec.from_sql(
+    """
+    CREATE TABLE myDeltaTableSpecTestDb{ID}.tbl
```

Comment:
what happens if the developer adds satements like "clustered by", "sorted by" etc?

Reply:
Not supported. I added a line in the documentation. ZORDER should be one of the first things I add after this, but is also currently unsupported.

Reply:
I created an issue #96